custom/inddex/couchdata.py

```diff
@@ -0,0 +1,150 @@
+from memoized import memoized
+
+from corehq.apps.fixtures.models import FixtureDataItem, FixtureDataType
+
+
+class CouchData:
+
+    def __init__(self, domain, tables=None):
+        if not isinstance(tables, tuple):
+            raise TypeError('\'tables\' must be a tuple')
+        self.domain = domain
+        self.tables = {}
+        for table in FixtureDataType.by_domain(domain=domain):
+            if (tables and table.tag in tables) or not tables:
+                self.tables[table.tag] = table.get_id
+
+    @memoized
+    def get_data_from_table(self, table_name=None, fields=None, values=None, as_dict=False):
```

Comment:
Can you add a docstring to this method?  What is `values`, other than a tuple?  It looks like a tuple whose elements are two-element tuples?  What do each of these represent?  If this is provided by callers, why does it need to be transformed at all?